packages/mobile/src/identity/verification.ts

```diff
@@ -193,7 +191,35 @@ export function* startVerification({ withoutRevealing }: StartVerificationAction
   yield put(refreshAllBalances())
 }
 
+export function* restartableVerification(initialWithoutRevealing: boolean) {
+  let isRestarted = false
+  while (true) {
+    const withoutRevealing = !isRestarted && initialWithoutRevealing
+    yield call(navigate, Screens.VerificationLoadingScreen, {
+      withoutRevealing,
+    })
+    yield put(resetVerification())
+    yield call(getConnectedAccount)
+    if (isRestarted || (yield select(isVerificationStateExpiredSelector))) {
+      yield call(fetchVerificationState)
+    }
+
+    const { verification, restart } = yield race({
+      verification: call(doVerificationFlow, withoutRevealing),
+      restart: take(Actions.RESEND_ATTESTATIONS),
```

Comment:
What analytics events should we add to best capture data on "restarts"? I think we'll definitely want to be able to track the efficacy of this feature moving forward

Reply:
There was no explicit requirement, so I will create one with the name which make sense :)

Reply:
Awesome, thanks! Will be great to have data on this